examples/src/guest-book-borsh.ts

```diff
@@ -0,0 +1,82 @@
+import { NearBindgen, near, call, view, Vector } from 'near-sdk-js';
+import * as borsh from 'borsh';
+
+@NearBindgen({
+  serializer(value) {
+    return borsh.serialize(schema, value);
```

Comment:
schema is not defined, would result in error

Reply:
schema is defined in the bottom